upgrade/1.2/scripts/k8s/blacklist-kernel-modules.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```

Comment:
If this is a new script, should this just be 2022?

Reply:
yes.  copy-pasta.  will fix